tracing/opentelemetry-tracing-sync/README.md

```diff
@@ -0,0 +1,159 @@
+# OpenTelemetry Sync Tracing for Tapir

```

Comment:
good docs, but it would be best to make this a docs page, probably a new page following https://github.com/softwaremill/tapir/blob/master/doc/server/observability.md in the menu (which is defined here: https://github.com/softwaremill/tapir/blob/master/doc/index.md)